examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -23,18 +23,24 @@
 #include "autopas/options/TraversalOption.h"
 #include "autopas/options/TuningStrategyOption.h"
 #include "autopas/utils/NumberSet.h"
-#include "src/Objects/CubeClosestPacked.h"
-#include "src/Objects/CubeGauss.h"
-#include "src/Objects/CubeGrid.h"
-#include "src/Objects/CubeUniform.h"
-#include "src/Objects/Sphere.h"
+#include "objects/CubeClosestPacked.h"
+#include "objects/CubeGauss.h"
+#include "objects/CubeGrid.h"
+#include "objects/CubeUniform.h"
+#include "objects/Sphere.h"
```

Comment:
```suggestion
#include "src/configuration/objects/CubeClosestPacked.h"
#include "src/configuration/objects/CubeGauss.h"
#include "src/configuration/objects/CubeGrid.h"
#include "src/configuration/objects/CubeUniform.h"
#include "src/configuration/objects/Sphere.h"
```

full include paths are a bit clearer